### DISCUSSION TITLE
[Question][Hw7] public test

### DISCUSSION THREAD

**Discussion by Yeonwoo-Nam** [prosyslab-classroom/cs524-program-analysis#341] [Q&A] [answered]
Name: Yeonwoo Nam

In hw7, I get "unsatisfiable" for given test/example1.ll.
But gradescope says that "Public Tests Failed!!"
Are there more public test than example1.11 
or the public test is for correctely compiled example1.ll?

**Comment by yeonhee-ryou** [comment]
Your custom test cases are considered as public tests too.

**Comment by Yeonwoo-Nam** [comment]
with the exact version submitted, I pass all the custom tests with "make test", 
where submission is graded as "Public Tests Failed!!".

**Comment by yeonhee-ryou** [comment, marked as answer]
I've checked your repository with the latest commit and "make test" failed.
(+ check the results of github action)
